scripts/generate_ssl_cert

```diff
@@ -7,17 +7,24 @@
 set -e
 FILE=$1
 CN=$2
+FORCE=$3
 set -eu
-if [ -z "${CN}" ] || [ -z "${FILE}" ]; then
-	echo "usage: $0 <certname> <cn>"
+
+if [ -z "${CN}" ] || [ "${CN}" = "--force" ] || [ -z "${FILE}" ] || [ "${FILE}" = "--force" ]; then
```

Comment:
It might be better to use `getopt(1)` for command line handling. https://www.shellscript.sh/tips/getopts/

Reply:
I'm working on removing the shell script with ocaml code, now that the branch is rebased on top of Ben's work with an ocaml binary for generating certs (now it simply calls the script)

Reply:
getopt doesn't seem to support positional arguments at all.

Reply:
Maybe it's not worth doing.
https://stackoverflow.com/questions/11742996/is-mixing-getopts-with-positional-parameters-possible